app/livechat/server/config.js

```diff
@@ -371,16 +371,16 @@ Meteor.startup(function() {
 			{ key: 'External', i18nLabel: 'External_Service' },
 			{ key: 'Least_Amount', i18nLabel: 'Least_Amount' },
 			{ key: 'Guest_Pool', i18nLabel: 'Guest_Pool' },
+			{ key: 'Load_Balancing', i18nLabel: 'Load_Balancing' },
 		],
 	});
 
-	settings.add('Livechat_guest_pool_with_no_agents', false, {
+	settings.add('Livechat_accept_chats_with_no_agents', false, {
```

Comment:
Did you update the i18n?

Reply:
It wasn't necessary because I'm using the same `i18nLabel` used by the previous setting.